accelerator/config.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
  * @brief Configuration of tangle-accelerator
  */
 
-#define TA_VERSION "tangle-accelerator/0.9.2"
+#define TA_VERSION "tangle-accelerator/0.9.3"
```

Comment:
Shall we bump version within this pull request?

Reply:
We bumped the version on branch `v0.9.x`.
I will remove the modification here.